src/Network/GRPC/MQTT/Client.hs

```diff
@@ -434,7 +436,7 @@ makeSessionIdTopic gen = do
 -- if the 'MethodName' provided would not form a valid MQTT topic.
 --
 -- >>> makeMethodRequestTopic "base.topic" "BgHY9DxnsLj7-IEq4IxTgqMg" "/proto.package.ServiceName/MyRPC"
--- Topic {unTopic = "base.topic/grpc/request/BgHY9DxnsLj7-IEq4IxTgqMg/proto.package.ServiceName/MyRPC"}
+-- Topic {unTopic = "base.topic/grpc/request/BgHY9DxnsLj7-IEq4IxTgqMg/proto-package-ServiceName/MyRPC"}
```

Comment:
Since the base topic is passed to the library by the users, I don't think we should change it. That should be the responsibility of the caller. What do you think?

Reply:
I don't think we should change the base topic. The user may have security policies set up that, for example, whitelist certain topics and block others. Changing the base topic behind the scenes could break that use case.